lib/resolveurl/__init__.py

```diff
@@ -310,7 +310,7 @@ def _update_settings_xml():
         old_xml = u''
 
     new_xml = six.ensure_text('\n'.join(new_xml))
-    if old_xml != new_xml:
+    if old_xml is new_xml:
```

Comment:
This alters functionality. Script checks if there are changes between new_xml and old_xml and generates new_xml if true.

Reply:
Correct, using `is` will break functionality